models/opensea/ethereum/opensea_v1_ethereum_events.sql

```diff
@@ -59,6 +60,7 @@ WHERE
 (addrs[3] = '0x5b3256965e7c3cf26e11fcaf296dfc8807c01073'
         OR addrs[10] = '0x5b3256965e7c3cf26e11fcaf296dfc8807c01073')
 AND call_success = true
+-- AND call_block_number = 12395486 -- for debugging
```

Comment:
Could you just remove those from the final PR 🙏 ?

Reply:
Sure, done 😎